post/index.md

```diff
@@ -0,0 +1,295 @@
+---
+title: "Mastering Directus: A Comprehensive Guide to Aggregation and Grouping Filters"
+description: "120-160 characters"
+author:
+    name: "jay bharadia"
+    avatar_file_name: "my-avatar.jpg"
+---
+
+## Introduction
+
+In the ever-evolving landscape of web development, the need for robust content management systems (CMS) has become more apparent than ever.
+
+-   Directus stands out as a powerful choice, offering a myriad of features to handle data intricacies.
+-   However, navigating the complexities of advanced filtering, especially with nesting and logical operators, can be a daunting task.
+
+Join us as we embark on a hands-on exploration, providing practical examples, insightful strategies, and a roadmap to elevate your Directus proficiency.
+
+Welcome to "Mastering Directus Advanced Filtering" – your comprehensive guide to navigating the depths of Directus with confidence and efficiency.
+
+## Before You Start
+
+> ℹ️ This guide uses the new [Directus SDK 11](https://docs.directus.io/guides/sdk/getting-started.html). Please refer docs if you are using an older version.
+
+### Consider we have Blog Website
+
+-   In our blog website, we have articles list with filters
+
+### Article List
+
+![Article List!](./images/all-articles-list.png)
+
+### Filters
+
+![Article List!](./images/filters.png)
+
+> ## Real World Requirement
+
+-   We need to show articles as per the **filters** are selected ( we have month , year filter ) and **search** and **sort**
+-   Get data Month wise , Year wise , also `when month and year both are selected`
+-   Get Data when search value is entered
+-   when combination of above is happened ( month + year + search )
+-   show Count of data as per the filters
+
+---
+
+### Directus Schema Setup
+
+-   Set up your schema as shown in screenshot
+
+![Blog Schema](./images/blog-schema.png)
+
+-   Now create `published_date` with `date` format
+
+![Article List!](./images/published-date-field.png)
+
+### Accessing Items ( JS Part)
+
+> Items are individual pieces of data in your database. They can be anything, from articles, to IoT status checks. Learn more about [Items](https://docs.directus.io/user-guide/overview/glossary.html#items).
```

Comment:
We can remove this because it's summarized in the short paragraph in the last suggestion.